### DISCUSSION TITLE
Google Play disable automatic release

### DISCUSSION THREAD

**Discussion by reixa00** [fastlane/fastlane#16954] [Q&A]
Hi all.

Is there a way to upload_to_play_store without automatic releasing the apk? Would enabling **Timed publishing** be enough?

Thanks!

**Comment by camilape** [comment]
@axierjhtjz, As far as I know, Fastlane doesn't have the Timed publishing feature still available, but you can control when do you want to release your apps deploying to another track i.e. Beta, and then promote your build to production

**Reply by rogerluan** [reply]
Yes ☝️ that's what I do :) I use the `alpha` track!

**Comment by sneko** [comment]
@axierjhtjz correct me if I'm wrong but it exists a parameter in the UI to turn this off.

![image](https://user-images.githubusercontent.com/8195958/130048484-7276f7b0-a5fa-42c8-9669-44c6eb21dc04.png)

**Reply by jenskuhrjorgensen** [reply]
@sneko Meaning it is a "global" setting and not a "per app" setting? So once "Managed publishing" is turned on, deploying apps directly to the "production" track with fastlane's `upload_to_play_store ` will in fact not release it to users but just send it for Google review, right? And then I can push the button to do the actual release when I want to.
